docs/content/contributing/developer-setup.md

```diff
@@ -21,14 +21,10 @@ The following environment variables are expected by the steps in this guide:
 
 Variable | Example | Description
 -------- | ------- | -----------
-`GOPATH` | $HOME/odev | Golang project directory
-`PGOROOT` | $GOPATH/src/github.com/crunchydata/postgres-operator | Operator repository location
+`PGOROOT` | $HOME/odev/postgres-operator | Operator repository location
 `PGO_CONF_DIR` | $PGOROOT/installers/ansible/roles/pgo-operator/files | Operator Config Template Directory
-`PGO_BASEOS` | {{< param centosBase >}} | Base OS for container images
 `PGO_CMD` | kubectl | Cluster management tool executable
-`PGO_IMAGE_PREFIX` | crunchydata | Container image prefix
 `PGO_OPERATOR_NAMESPACE` | pgo | Kubernetes namespace for the operator
```

Comment:
Was just curious why we're removing some of these (`PGO_BASEOS`, `PGO_OPERATOR_NAMESPACE` & `PGO_VERSION`) and not the others.  All of these appear to be defined in the default Makefile, so just wondering which we're keeping in this list.

Also, would `PGO_OPERATOR_NAMESPACE` be more Bash install thing vs. dev setup?  I do see we have a `Deployment` section in the instructions that references what is essentially the Bash install, which would require this env var.  Maybe we should just point to the bash install instructions instead (especially considering the various other setup, e.g. for things like RBAC, CRD's, etc, are really needed to properly deploy)?

Reply:
> Was just curious why we're removing some of these (`PGO_BASEOS`, `PGO_OPERATOR_NAMESPACE` & `PGO_VERSION`) and not the others. All of these appear to be defined in the default Makefile, so just wondering which we're keeping in this list.

I intended to remove only the variables related to building images because those are only used in the Makefile. However, I see that `PGO_IMAGE_PREFIX` is also used in `bin/push-to-gcr.sh`.

> Also, would `PGO_OPERATOR_NAMESPACE` be more Bash install thing vs. dev setup? I do see we have a `Deployment` section in the instructions that references what is essentially the Bash install, which would require this env var. Maybe we should just point to the bash install instructions instead (especially considering the various other setup, e.g. for things like RBAC, CRD's, etc, are really needed to properly deploy)?

I'm fine with linking to the Bash install docs for the Deployment section. I really just punted on figuring out when/why `PGO_CMD` and `PGO_OPERATOR_NAMESPACE` should be set because they are used throughout the `deploy` directory.